ukis_pysat/raster.py

```diff
@@ -73,6 +73,15 @@ def arr(self):
         else:
             return reshape_as_image(self.__arr)
 
+    @arr.setter
+    def arr(self, value):
+        """array setter"""
+
+        if not isinstance(value, int) and not isinstance(value, np.ndarray):

```

Comment:
it should always be an array of the same shape, when you make it an int or an array of a different shape we're getting into troubles.

Reply:
True. So you suggest to also check for the same shape?

Reply:
Yes, and what is the idea behind checking for type int?

Reply:
My idea was that an array can also be represented by a scalar, thus, 1-dimensional.

Reply:
I think this is not really a valid use case for us.

Reply:
Agree with @fwfichtner to check for array shape in xy dimension. z dimension does not affect geotransformation and therefore can vary.